.env

```diff
@@ -0,0 +1,15 @@
+### Default Environment Variables
+## General
+ENV_ENVTEST_K8S_VERSION=1.27.1 # ENVTEST_K8S_VERSION refers to the version of Kubebuilder assets to be downloaded by envtest binary.
+ENV_GARDENER_K8S_VERSION=1.27
+ENV_IMG=europe-docker.pkg.dev/kyma-project/prod/telemetry-manager:main # Image URL to use all building/pushing image targets
```

Comment:
Update the release process [here](https://github.com/kyma-project/telemetry-manager/blob/main/docs/contributor/releasing.md#release-process). In step 5, we need now to reference the new  `ENV_IMG` variable in the `.env` file instead of the old `IMG` variable in `Makefile`